deltachat-ios/Coordinator/AppCoordinator.swift

```diff
@@ -107,6 +107,9 @@ class AppCoordinator: NSObject, Coordinator {
 	func presentLoginController() {
 		let accountSetupController = AccountSetupController()
 		let accountSetupNav = DCNavigationController(rootViewController: accountSetupController)
+    let coordinator = AccountSetupCoordinator(navigationController: accountSetupNav)
```

Comment:
seems as if the file is indented with tabs and the pr uses spaces.
maybe we should switch to space-indentation at some point ...